R/opart_gaussian.R

```diff
@@ -10,7 +10,7 @@
 #' opart_gaussian(c(1,2,3,4), 2)
 
 
-opart_gaussian <- structure(function(data, penalty) {
+opart_gaussian <- function(data, penalty) {
```

Comment:
@tdhock , 

Does this look fine?

Reply:
`data` is a base R function for loading a data set, so please change the name of this argument to something else -- maybe `data.vec` to suggest that it should be a vector?